ci-operator/step-registry/openstack/deprovision/bastionproxy/openstack-deprovision-bastionproxy-commands.sh

```diff
@@ -11,6 +11,30 @@ fi
 
 export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml
 CLUSTER_NAME=$(<"${SHARED_DIR}"/CLUSTER_NAME)
+BASTION_FIP=$(<"${SHARED_DIR}"/BASTION_FIP)
+
+# configure the local container environment to have the correct SSH configuration
+if ! whoami &> /dev/null; then
+    if [[ -w /etc/passwd ]]; then
+        echo "${BASTION_USER}:x:$(id -u):0:${BASTION_USER:-centos} user:${HOME}:/sbin/nologin" >> /etc/passwd
```

Comment:
The step openstack-deprovision-deletefips happens before openstack-deprovision-bastionproxy, consequently bastion fip will be gone once the logs are attempted to be collected